python/ray/tests/test_runtime_env_2.py

```diff
@@ -240,7 +240,9 @@ def job_failed(job_id):
         with pytest.raises(AssertionError):
             assert_no_user_info_in_logs(USER_SECRET)
 
-        assert_no_user_info_in_logs(USER_SECRET, file_whitelist=["runtime_env*.log"])
+        assert_no_user_info_in_logs(
+            USER_SECRET, file_whitelist=["runtime_env*.log", "event_EXPORT*.log"]
```

Comment:
why is this necessary? There shouldn't be "user info" in the export events is there?

Reply:
Various export events contain runtime env info (which is considered user info) because the runtime env is returned in some dashboard APIs